extensions/package-vulnerability-scanner/manifest.json

```diff
@@ -37,15 +37,15 @@
 		}
 	},
 	"extension": {
-		"name": "publisher-cves",
-		"title": "Publisher CVEs",
-		"description": "A dashboard for publishers to see vulnerabilities in their content.",
-		"homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/publisher-cves",
+		"name": "package-vulnerability-scanner",
+    "title": "Package Vulnerability Scanner",
+    "description": "Scan for security vulnerabilities in your published content on Connect using Posit Package Manager. View an overview of CVEs (Common Vulnerabilities and Exposures) across all your apps, and get guidance on upgrading affected packages.",
```

Comment:
Big part of the PR here. This is what will show up in the Connect Gallery for this extension